scripts/ethbuild.sh

```diff
@@ -223,7 +238,12 @@ do
 			exit 1
 		fi
 
-		make -j${MAKE_CORES}
+		if [[ $USEXCODE -eq 0 ]]; then
+			make -j${MAKE_CORES}
+		else
+			xcodebuild
```

Comment:
That condition wasn't there last time I checked this PR. If you put it under a special argument then it should be fine.